Automattic/jetpack
# CRM: Add new Support Contact page

## Proposed changes:


This PR adds the new Support Contact page. If the user has a valid license, they can send a support ticket directly to Zendesk.

**(1) Without license:**

![image](https://user-images.githubusercontent.com/9832440/226047483-8559e2d5-9534-493d-adfa-4903d24d7e46.png)

**(2) With an invalid license:**

![image](https://user-images.githubusercontent.com/9832440/226047131-6ff1a3c7-0969-4a50-8b12-c6bb79cab233.png)

**(3)With a valid license:**

![image](https://user-images.githubusercontent.com/9832440/226050432-4783c9ba-591b-4723-b538-7ed2638cab1e.png)

If the user has a license, this `Support` menu is added:

![image](https://user-images.githubusercontent.com/9832440/226091739-3911b34d-788e-40c1-9881-aa018f9f08b5.png)

Also, the top JPCRM menu was updated for any user:

![image](https://user-images.githubusercontent.com/9832440/226091809-dfbd731b-ee77-46f8-9c07-09cc40a38015.png)

**Actions available**
- Click on 'Support` WP menu or JPCRM top menu will send you to the Support page `wp-admin/admin.php?page=jpcrm-support`
- Click on `Knowledge Base` will send you to https://kb.jetpackcrm.com page (new tab)
- Click on `Support Forum` will send you to https://wordpress.org/support/plugin/zero-bs-crm (new tab)
- With a valid license, it'll show you the support contact form. Filling it out, and clicking on `Submit` will create a ticket on Zendesk with a summary of technical data from the site.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:

( [Go to the issue](https://github.com/Automattic/zero-bs-crm/issues/2983) to see the instructions there )
